docs/release-notes/8.14.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[release-notes-header-8.14.0]]
+== 8.14
+
+[discrete]
+[[release-notes-8.14.0]]
+=== 8.14.0
+
+[discrete]
+[[features-8.14.0]]
+==== New features
+* Introduces Attack discovery, a new feature that uses AI to identify potential attacks and help you quickly triage multiple alerts ({kibana-pull}181818[#181818]).
+* Creates the **Asset criticality** page within the **Manage** menu, which lets you assign asset criticality levels to your assets ({kibana-pull}179891[#179891]).
+* Enables alert suppression for New Terms rules ({kibana-pull}178294[#178294]).
+* Enables alert suppression for rules with non-sequence EQL queries ({kibana-pull}176422[#176422]).
+* Adds a **Setup guide** markdown field to custom rules ({kibana-pull}178131[#178131]).
+
+[discrete]
+[[enhancements-8.14.0]]
+==== Enhancements
+* Removes the "Technical preview" tag for {esql}; it is now GA ({kibana-pull}180838[#180838]).
+* Allows you to add newly created fields to an {esql} rule's highlighted fields ({kibana-pull}177746[#177746]).
+* Connects {esql} functionality in {elastic-sec} to the `discover:enableESQL` advanced setting ({kibana-pull}181616[#181616]).
+* Makes conversations with Elastic AI Assistant persist across sessions ({kibana-pull}173487[#173487]).
+* Adds conversation streaming for Elastic AI Assistant ({kibana-pull}180095[#180095]).
+* Adds support for Anthropic Claude 3 to the Amazon Bedrock connector, and makes it the default model ({kibana-pull}179304[#179304]).
+* Adds an **AI Assistant** settings section to the **Management** menu ({kibana-pull}176656[#176656]).
+* Adds the `_source` field to the Alert details flyout's JSON view ({kibana-pull}180477[#180477]).
```

Comment:
@PhilippeOberti how does this improve the user's experience with the JSON tab in the alert details flyout? I vaguely remember this conversation in one our Slack threads, but could use a refresher. Thanks! 

```suggestion
* Adds the `_source` field to the alert details flyout's JSON view ({kibana-pull}180477[#180477]).
```

Reply:
some people were missing it, and pointing that Discover has it, so in a sense yes it does improve the experience for some users!